mlir/lib/Dialect/Affine/Analysis/LoopAnalysis.cpp

```diff
@@ -212,6 +214,41 @@ void mlir::affine::getTripCountMapAndOperands(
                             tripCountValueMap.getOperands().end());
 }
 
+/// Take the min if all trip counts are constant.
+static std::optional<uint64_t>
+getConstantTripCountFromAffineMap(AffineMap map,
```

Comment:
This at the very least feels like it should be named something like `getKnownTripCountBound`